doc/classes/Dictionary.xml

```diff
@@ -5,7 +5,7 @@
 	</brief_description>
 	<description>
 		Dictionaries are associative containers that contain values referenced by unique keys. Dictionaries will preserve the insertion order when adding new entries. In other programming languages, this data structure is often referred to as a hash map or an associative array.
-		You can define a dictionary by placing a comma-separated list of [code]key: value[/code] pairs inside curly braces [code]{}[/code].
+		You can define a dictionary by placing a comma-separated list of [code]key: value[/code] pairs inside curly braces [code]{}[/code]. Duplicate dictionary keys are not allowed during declaration.
```

Comment:
Is this remark necessary? It is already established in one word prior ("**unique** keys") that duplicate keys are not allowed. Generally, not just during declaration.
```suggestion
		You can define a dictionary by placing a comma-separated list of [code]key: value[/code] pairs inside curly braces [code]{}[/code].
```
If we really have to it can be briefer, at least:
```suggestion
		You can define a dictionary by placing a comma-separated list of [code]key: value[/code] pairs inside curly braces [code]{}[/code]. Duplicate keys are not allowed.
```